dashboard/app/views/plc/user_course_enrollments/index.html.haml

```diff
@@ -21,17 +21,17 @@
               - practice_module_assignment = unit_enrollment.module_assignment_for_type Plc::LearningModule::PRACTICE_MODULE
               = link_to script_path(unit_enrollment.plc_course_unit.script, anchor: 'overview') do
                 .module_assignment
-                  Overview
+                  = t('flex_category.overview')
```

Comment:
Hmm - actually maybe these should be overview instead of required. Things aren't different right now so I might as well keep it that way